samples/openthread/libraries/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+cmake_minimum_required(VERSION 3.13.1)
+
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
```

Comment:
shouldn't this use find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})??

Reply:
I used [nrf_security lib CMakeListst.txt](https://github.com/piotrkoziar/fw-nrfconnect-nrf/blob/ot-nrf_security-ncs/samples/openthread_cc310_lib_build/CMakeLists.txt) as a reference.

Is there any benefit in changing it to the `find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})`.
To be honest I do not understand the magic behind this line yet.

Reply:
I do not as well:) but AFAIR the find_package is the new approach and I was sure all manual includes were replaced long time ago (seems I was mistaken)

Reply:
It turns out that find_package is recommended because it provides better build information error in case if the environment variable is missing. Look at [this](https://github.com/zephyrproject-rtos/zephyr/pull/25808)